server/php/UploadHandler.php

```diff
@@ -377,7 +377,11 @@ protected function get_error_message($error) {
     public function get_config_bytes($val) {
         $val = trim($val);
         $last = strtolower($val[strlen($val)-1]);
-        $val = (int)substr($val, 0, -1);
+        if (false == preg_match("/[0-9]$/", $val)) {
```

Comment:
I think this can be improved by using the built-in `is_numeric` function:
```php
if (is_numeric($last)) {
    $val = (int)$val;
} else {
    $val = (int)substr($val, 0, -1);
} 
```

Reply:
True, I did have some problems with is_numeric in the past, but that was a long time ago, admittedly. It seems to do it's job nowadays. :)